src/modules/previewpane/UnitTests-SvgPreviewHandler/SvgPreviewControlTests.cs

```diff
@@ -93,61 +94,16 @@ public void SvgPreviewControl_ShouldSetScrollBarsEnabledProperty_WhenDoPreviewCa
             Assert.AreEqual(((WebBrowser)svgPreviewControl.Controls[0]).ScrollBarsEnabled, true);
         }
 
-        [TestMethod]
-        public void SvgPreviewControl_ShouldAddTextBox_IfBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));
-
-            // Assert
-            Assert.IsInstanceOfType(svgPreviewControl.Controls[0], typeof(RichTextBox));
-            Assert.IsInstanceOfType(svgPreviewControl.Controls[1], typeof(WebBrowser));
-            Assert.AreEqual(svgPreviewControl.Controls.Count, 2);
-        }
-
-        [TestMethod]
-        public void SvgPreviewControl_ShouldNotAddTextBox_IfNoBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>valid</svg>"));
-
-            // Assert
-            Assert.IsNotInstanceOfType(svgPreviewControl.Controls[0], typeof(RichTextBox));
-            Assert.AreEqual(svgPreviewControl.Controls.Count, 1);
-        }
-
-        [TestMethod]
-        public void SvgPreviewControl_ShouldAddValidTextBox_IfBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));
-            var textBox = svgPreviewControl.Controls[0] as RichTextBox;
-
-            // Assert
-            Assert.IsFalse(string.IsNullOrWhiteSpace(textBox.Text));
-            Assert.AreEqual(textBox.Dock, DockStyle.Top);
-            Assert.AreEqual(textBox.BackColor, Color.LightYellow);
-            Assert.IsTrue(textBox.Multiline);
-            Assert.IsTrue(textBox.ReadOnly);
-            Assert.AreEqual(textBox.ScrollBars, RichTextBoxScrollBars.None);
-            Assert.AreEqual(textBox.BorderStyle, BorderStyle.None);
-        }
-
         [TestMethod]
         public void SvgPreviewControl_RichTextBoxWidthShouldAdjust_IfParentControlWidthChanges()
         {
             // Arrange
             var svgPreviewControl = new SvgPreviewControl();
-            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));
+            var mockStream = new Mock<IStream>();
+            mockStream

```

Comment:
I don't really understand what this mock does or why it needs to throw, to satisfy the test?

Reply:
The test is for if a info bar is added then its  width should adjust as with the parent form window. Currently an info bar is only added if the preview of Svg throws. Throwing the stream is one to achieve that. Updated the test description that it's for the case when preview throws.